hera_mc/tests/test_qm.py

```diff
@@ -0,0 +1,132 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Testing for `hera_mc.qm`.
+
+"""
+import unittest
+
+import numpy as np
+from math import floor
+from astropy.time import Time, TimeDelta
+
+from hera_mc import mc, cm_transfer
+from hera_mc.qm import ant_metrics, array_metrics, metric_list
+from hera_mc import utils
+from hera_mc.tests import TestHERAMC
+
+
+class TestQM(TestHERAMC):
+
+    def setUp(self):
+        super(TestQM, self).setUp()
+        t1 = Time('2016-01-10 01:15:23', scale='utc')
+        t2 = t1 + TimeDelta(120.0, format='sec')
+        self.obsid = utils.calculate_obsid(t1)
+        # Create obs to satifsy foreign key constraints
+        self.test_session.add_obs(t1, t2, self.obsid)
```

Comment:
We recently learned that if you have things that fail in the setup that it makes nosetests hang rather than just returning an error. So I suggest copying the parts where you actually add things to the session into the tests. It's annoying because it duplicates code, but nosetests hanging is really awful.